ppr-ui/src/components/mhrRegistration/ReviewConfirm/YourHomeReview.vue

```diff
@@ -179,6 +179,16 @@ export default defineComponent({
       }),
       engineerDisplayDate: computed((): string => {
         return yyyyMmDdToPacificDate(getMhrRegistrationHomeDescription.value?.engineerReportDate, true)
+      }),
+      manufactureYear: computed((): string => {
```

Comment:
typo in `manufacturer`? 
Or maybe not, unless that is intentional. Could be read both ways i suppose!

Reply:
this is because the text field is called "Year of Manufacture", so I call it "manufactureYear"

Reply:
Makes sense, thanks for checking!